courseware/intro_data_science.md

```diff
@@ -40,7 +40,7 @@ As an example, MIT's large [Introduction to Machine Learning](https://introml.od
 
 We strongly encourage you to change and adapt these notebooks to fit your needs! You can download any notebook in either .ipynb or .py format by clicking on its link and File > Download.
 
-Datasets can be changed by editing the list of variables queried (see the [“Data Commons for Data Science”](https://colab.research.google.com/drive/1FUQGfoIFr9vwa1wRTmVXVdTYHwr7ntHM) tutorial for more on this); editing framing and questions is as easy as editing text cells.
+Datasets can be changed by editing the list of variables queried (see the [“Data Commons for Data Science”](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/intro_data_science/Data_Commons_For_Data_Science_Tutorial.ipynb) tutorial for more on this); editing framing and questions is as easy as editing text cells.
```

Comment:
Sorry for the late comment.  The `{:target="_blank"}` in the other page is nice.  Is it worth adding here?